### ISSUE TITLE
S Video Trim is worse than useless

### ISSUE THREAD

**Issue by Rudxain** [0x192/universal-android-debloater#861]
**Your phone model**: SM-A315G

**Packages documentation to update:**

```
com.samsung.app.newtrim
```

### com.samsung.app.newtrim
**Removal**: `Advanced`
:arrow_right: `Recommended`

### New name

Editor Lite

### etc

This trimmer is both imprecise and inaccurate, `ffmpeg` is much better.

**Comment by AnonymousWP** [comment]
It's not bloatware, but has a useful function, so I think `Advanced` fits better.

**Comment by Rudxain** [comment]
@AnonymousWP 
> It's not bloatware, but has a useful function, so I think `Advanced` fits better.

It would be useful, if it weren't for the fact that its only "feature" isn't done properly.

On my device (and, IIRC, also the SM-J327VPP) trimming from either end is inaccurate by multiple frames, **especially the ending**. Trimming from the last-end is **inaccurate by 2 or more seconds**, and it depends on the direction on which the "handle" was last slided by my finger (what?!)

This app literally had one job, and it does it poorly. It doesn't even deserve to exist.

The only reason not to use FFMPEG as an alternative, is because the app has a preview GUI, but even that is inaccurate...

It's like [Samsung devs are underpaid, burnt-out, or just incompetent](/Rudxain/Samsung-rants)

**Comment by AnonymousWP** [comment]
Yeah I understand what you mean, but what if this is a temporary bug/issue and in the future it's working as intended? But yeah, if there's literally no reason to use S Video Trim instead of FFMPEG then I agree with setting it to `Recommended`.

And lol at the Samsung dev GitHub page, lmao.

**Comment by Rudxain** [comment]
@AnonymousWP
> what if this is a temporary bug/issue and in the future it's working as intended?

I agree. It's possible they've fixed it in One UI v6.0, I'm on v4.1.

> And lol at the Samsung dev GitHub page, lmao.

Thank you for reading the page, lol. It's getting kinda long, so I might split it in "category pages"

**Comment by AnonymousWP** [comment]
> @AnonymousWP
> 
> > what if this is a temporary bug/issue and in the future it's working as intended?
> 
> I agree. It's possible they've fixed it in One UI v6.0, I'm on v4.1.

Could you try?

**Comment by Rudxain** [comment]
> Could you try?

Samsung decided to stop sending updates to A31, so I can't test it... unless I try to install the newest video-trim from a 3rd-party source
